client/my-sites/upgrades/trials/start-trial/terms-of-service.jsx

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
I have just changed `By checking out` to `By continuing` since we were trying to avoid showing anything related to a purchase. We could definitively use `checkout/terms-of-service.jsx` to avoid code duplication if you think it's not worth it.